src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -19,7 +19,6 @@
  * Supports a minimal set of list operations.
  *
  * @see Person#equals(Object)
- * @see CollectionUtil#elementsAreUnique(Collection)
```

Comment:
Accidental removal?

Reply:
Cannot resolve collections and the line will be underlined red

Reply:
Lol it isn't the highlight function. If you take a closer look and trace the code, it is because `UniquePersonList.java` does not have the import statement `import java.util.Collection;`. If you add that in the redness will go away, but since it is still technically a JavaDoc comment I suppose it does not have practical impacts on the application